### ISSUE TITLE
paper-checkbox accessibility issues 

### ISSUE THREAD

**Issue by lpalmaro** [PolymerElements/paper-checkbox#11]
Here are the a11y developer tools audit results: 

[Warning] Text elements should have a reasonable contrast ratio (1)
See AX_COLOR_01 for more information.
Not applicable tests (7)
Passing tests (7)

For the most part, the contrast looks good to me. The only item that is too light is the disabled unchecked checkbox, which I understand is light intentionally. If we can make that just slightly more visible, that would be great.

**Comment by alice** [comment]
User-provided `role` is overridden. This is a common issue; suggest creating a helper function in library for setting default ARIA role/attributes which respects user-provided values.

**Comment by notwaldorf** [comment]
Based on the `paper-radio-button` conversation, here's what it looks like after applying the same values as there:

![screen shot 2015-05-19 at 11 46 04 am](https://cloud.githubusercontent.com/assets/1369170/7711022/b89e9062-fe1c-11e4-9c60-331b7f02d879.png)

**Comment by lpalmaro** [comment]
This looks good. I have to be honest that I prefer the gray focus ripple to
be slightly darker still, as it's still a bit difficult to see... but other
than that, this is great.

**Comment by notwaldorf** [comment]
Fixed with https://github.com/PolymerElements/paper-checkbox/pull/17

**Comment by alice** [comment]
This isn't fixed, please re-open and address my comment above.

**Comment by notwaldorf** [comment]
@alice do you mean it's not fixed on the demo site, or locally? I don't think the demo site has been updated yet..

**Comment by alice** [comment]
Apologies - as discussed elsewhere, I was mistaken; this is actually fixed.

**Comment by lpalmaro** [comment]
Is this good to close out?

**Comment by notwaldorf** [comment]
It's already closed :)

**Comment by lpalmaro** [comment]
Ah, sorry! Thanks! :)

On Tue, May 26, 2015 at 3:36 PM, Monica Dinculescu <notifications@github.com
